app/indexers/cdl_resource_indexer.rb

```diff
@@ -7,10 +7,10 @@ class CdlResourceIndexer < Hyrax::Indexers::PcdmObjectIndexer(CdlResource)
   include Hyrax::Indexer(:cdl_resource)
 
   # Uncomment this block if you want to add custom indexing behavior:
-  #  def to_solr
-  #    super.tap do |index_document|
-  #      index_document[:my_field_tesim]   = resource.my_field.map(&:to_s)
-  #      index_document[:other_field_ssim] = resource.other_field
-  #    end
-  #  end
+  def to_solr
+    super.tap do |index_document|
+      index_document['admin_note_tesim'] = resource.admin_note
```

Comment:
i actually don't think we need any of the `index_document['admin_note_tesim']` indexed explicitly, I _believe_ the .yaml should take care of it., might be good to check though 🤔

Reply:
oh man, i didn't even notice the yaml files. 👀

Reply:
it uses the metadata yml files like the models do? @kirkkwang

Reply:
In that case, i don't really think we need this PR 😭 Thanks for pointing that out, too bad I didn't notice that sooner.